sites/partners/cypress/fixtures/application.json

```diff
@@ -34,8 +34,8 @@
   "householdStudent": "No",
   "incomePeriod": "Month",
   "incomeMonth": "5000",
-  "incomeVouchers": "No",
-  "demographics.ethnicity": "Not Hispanic / Latino",
+  "incomeVouchers": "None of the above",
```

Comment:
I'm surprised to see the use of "None of the above" or "n/a" in other cases as values for incomeVouchers when it seems that the keys only include "none". Is this accidental?